backend/fms_core/templates.py

```diff
@@ -489,7 +489,25 @@
       },
   ],
   "user prefill info": {
+      # "Extraction Type": ["DNA", "RNA"], # already prefilled by workflow
+      "Volume Used (uL)": "number",
+      # borrowed from extraction template
+      "Destination Container Kind": [
```

Comment:
you should cast SAMPLE_CONTAINER_KINDS from containers.py into a list instead. The less places we have to update the same information the best.

Reply:
i'm not quite sure how it applies in here. i picked only the containers that are relevant to sample extraction. is this just for improving existing codes?

Reply:
It applies that w can typically extract in any container that can receive samples. These are listed in the above mentioned constant. This is meant to make the code require less maintenance. This list will be automatically updated with new values as new containers are created while your hard coded list will need to be updated specifically each time.